### ISSUE TITLE
NAs in @summaryresults

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/doppelgangR#10]
This was originally by design to avoid storing all correlations until wrap-up.  See if it is easy to resolve, would be better to fill all columns in all rows of @summaryresults.

**Comment by lwaldron** [comment]
I hoped that by re-working the smoking gun finder, this and the repetition issue would just get fixed, but they didn't.

**Comment by lwaldron** [comment]
Still happens, here's an example.  I guess this could be called an enhancement though.

``` r
library(curatedOvarianData)
library(doppelgangR)
data("GSE32063_eset")
data("GSE6008_eset")
esets <- list(GSE32063=GSE32063_eset, GSE6008=GSE6008_eset)
testdop <- doppelgangR(esets)
edit(testdop@summaryresults[, 1:9])
```

**Comment by lwaldron** [comment]
Resolved in 0.10.0 and up.
